src/V3ExecGraph.cpp

```diff
@@ -396,7 +403,7 @@ class PackThreads final {
 
             // Populate busyUntil timestamps. For multi-worker tasks, set timestamps for
             // offsetted threads.
```

Comment:
With the broadcasting of `scuedule.endTime()` to  of all values of `busyUntil` in the mode switches you added above, is the `for` loop in the `else` here still necessary? (Within the same schedule, you only consider the threads of that schedule - I hope - so you should never read `busyUntil` indices higher then the current schedule.m_nThreads, then the broadcasting above fixes up when you start a new schedule).

Reply:
You're right, removed.
This part will become relevant when interleaving of single and multi-threaded tasks will be implemented.